x-pack/test/cases_api_integration/security_and_spaces/tests/common/user_actions/migrations.ts

```diff
@@ -1046,6 +1052,76 @@ export default function createGetTests({ getService }: FtrProviderContext) {
         ]);
       });
     });
+
+    describe('8.3.0', () => {
+      const CASE_ID = '5257a000-5e7d-11ec-9ee9-cd64f0b77b3c';
+
+      before(async () => {
+        await kibanaServer.importExport.load(
+          'x-pack/test/functional/fixtures/kbn_archiver/cases/8.0.0/cases.json'
+        );
+      });
+
+      after(async () => {
+        await kibanaServer.importExport.unload(
+          'x-pack/test/functional/fixtures/kbn_archiver/cases/8.0.0/cases.json'
+        );
+        await deleteAllCaseItems(es);
+      });
+
+      describe('add severity', () => {
+        it('adds the severity field to the create case user action', async () => {
+          const userActions = await getCaseUserActions({
+            supertest,
+            caseID: CASE_ID,
+          });
+
+          const createUserAction = userActions[0];
+          expect(createUserAction).to.eql({
+            action: 'create',
+            action_id: '5275af50-5e7d-11ec-9ee9-cd64f0b77b3c',
+            case_id: '5257a000-5e7d-11ec-9ee9-cd64f0b77b3c',
+            comment_id: null,
+            created_at: '2021-12-16T14:34:48.709Z',
+            created_by: {
+              email: '',
+              full_name: '',
+              username: 'elastic',
+            },
+            owner: 'securitySolution',
+            payload: {
+              connector: {
+                fields: null,
+                id: 'none',
+                name: 'none',
+                type: '.none',
+              },
+              description: 'migrating user actions',
+              settings: {
+                syncAlerts: true,
+              },
+              status: 'open',
+              tags: ['user', 'actions'],
+              title: 'User actions',
+              owner: 'securitySolution',
+              severity: 'low',
+            },
+            type: 'create_case',
+          });
+        });
+
+        it('does NOT add the severity field to the other user actions', async () => {
+          const userActions = await getCaseUserActions({
+            supertest,
+            caseID: CASE_ID,
+          });
+
+          for (const userAction of userActions.slice(1)) {
```

Comment:
nit: Maybe instead of doing a slice could we do a filter to explicitly remove the `create_case` user action? I suppose it's unlikely but if we add create_case user actions for some reason it wouldn't break this test.